bin/generate.py

```diff
@@ -279,9 +279,9 @@ def generate_workbench_panels(response_tasks, stories, TEMPLATE_PATH, OUTPUT_PAT
                 template = j2_env.get_template('panel.j2')
                 file_path = "default/data/ui/panels/workbench_panel_" + response_file_name + ".xml"
                 output_path = path.join(OUTPUT_PATH, file_path)
-                if response_task['search'].find(">") is not -1:
+                if response_task['search'].find(">") != -1:
                     response_task['search']= response_task['search'].replace(">","&gt;")
```

Comment:
I think the "if"'s here are redundant. 
```
response_task['search']= response_task['search'].replace(">","&gt;")
```
is going to replace ">" if it exists and leave it untouched if it doesn't.

Reply:
oh good point 😄 fixing.